### PR TITLE
Account modal design improvements

### LINKED ISSUE #6972: Account modals (change password and email) do not match designs
**Designs:**
https://zpl.io/bWy9ld1
https://zpl.io/aMXz6l7

image

vs

image

- [ ] Insufficient spacing below title on modals - from #6971

### PR DESCRIPTION
**Pull request by davidpmccormick**
Closes #6972 

- adds password rules validating checklist
- uses Wellcome style for inputs
- uses Wellcome style for buttons
- tidies spacing

__TODO__
- [x] Fix failing tests

__Change password__
Screenshot 2021-09-23 at 16 59 04

__Change email__
Screenshot 2021-09-23 at 16 58 36

### CODE REVIEW

**Comment by DominiqueMarshall** [comment]
All good with me!

**Comment by davidpmccormick** [comment]
I've skipped 9 of the 88 tests – I wasn't able to work out why they were failing. I'd appreciate any ideas or help. I don't think they're critical to be in for the testing to start, so don't believe they need to block this pr though.

File: identity/webapp/src/frontend/hooks/usePasswordRules.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+const usePasswordRules = (
+  input: string
+): {
+  isAtLeast8Characters: boolean;
+  hasLowercaseLetters: boolean;
+  hasUppercaseLetters: boolean;
+  hasNumbers: boolean;
+} => {
```
Comment (jamesgorrie) [inline identity/webapp/src/frontend/hooks/usePasswordRules.tsx:10]: Can we set this as a type for nice-to-readness?

**Review by jamesgorrie — DISMISSED** [review]
Nice!

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**